source/NVDAObjects/window/winword.py

```diff
@@ -1430,10 +1466,520 @@ def _get_WinwordWindowObject(self):
+				"kb:enter": "activatePosition",
+				"kb(desktop):numpadEnter":"activatePosition",
+				"kb:space": "activatePosition",
+				"kb:escape": "disablePassThrough",
+	}
+
+class WordChartSeriesTrendline(WordChart):
+	_trendlineTypeMap = {_chartConstants.xlExponential: 'Exponential',
+								_chartConstants.xlLinear: 'Linear',
+								_chartConstants.xlLogarithmic: 'Logarithmic',
+								_chartConstants.xlMovingAvg: 'Moving Average',
+								_chartConstants.xlPolynomial: 'Polynomial',
+								_chartConstants.xlPower: 'Power' 
+	}
+
+	def __init__(self, windowHandle, wordApplicationObject, wordChartObject, keyIndex, seriesIndex, trendlineIndex=1):
+		self.windowHandle=windowHandle
+		self.wordApplicationObject=wordApplicationObject
+		self.seriesIndex=seriesIndex
+		self.trendlineIndex=trendlineIndex
+		self.trendlinesCount=wordChartObject.SeriesCollection(self.seriesIndex).Trendlines().Count
+		super(WordChartSeriesTrendline, self).__init__(windowHandle=windowHandle, wordApplicationObject=wordApplicationObject, wordChartObject=wordChartObject, keyIndex=keyIndex)
+
+	def _get_name(self):
+		if self.wordChartObject.SeriesCollection(self.seriesIndex).Trendlines(self.trendlineIndex).DisplayEquation or self.wordChartObject.SeriesCollection(self.seriesIndex).Trendlines(self.trendlineIndex).DisplayRSquared:
+			label=self.wordChartObject.SeriesCollection(self.seriesIndex).Trendlines(self.trendlineIndex).DataLabel.Text
+			#Translators: Substitute superscript two by square for R square value
+			label=label.replace(u"²", _( " square " ))
+			label=re.sub(r'([a-zA-Z]+)([2])',r'\1 square', label)
+			label=re.sub(r'([a-zA-Z]+)([3])',r'\1 cube', label)
+			label=re.sub(r'([a-zA-Z]+)([-]*[04-9][0-9]*)',r'\1 to the power \2', label)
+			#Translators: Substitute - by minus in trendline equations.
+			label=label.replace(u"-",_(" minus "))
+			# Translators: This message gives trendline type and name for selected series
+			output=_("{seriesName} trendline type: {trendlineType}, name: {trendlineName}, label: {trendlineLabel} ").format(seriesName=self.wordChartObject.SeriesCollection(self.seriesIndex).Name, trendlineType=self._trendlineTypeMap[self.wordChartObject.SeriesCollection(self.seriesIndex).Trendlines(self.trendlineIndex).Type], trendlineName=self.wordChartObject.SeriesCollection(self.seriesIndex).Trendlines(self.trendlineIndex).Name, trendlineLabel=label)
+		else:
+			# Translators: This message gives trendline type and name for selected series
+			output=_("{seriesName} trendline type: {trendlineType}, name: {trendlineName} ").format(seriesName=self.wordChartObject.SeriesCollection(self.seriesIndex).Name, trendlineType=self._trendlineTypeMap[self.wordChartObject.SeriesCollection(self.seriesIndex).Trendlines(self.trendlineIndex).Type], trendlineName=self.wordChartObject.SeriesCollection(self.seriesIndex).Trendlines(self.trendlineIndex).Name)
+		return output
+
+	def _get_role(self):
+		return controlTypes.ROLE_CHARTELEMENT

```

Comment:
Sure, it has been changed.
Changed code is in branch in_t1541_Fin
Thanks a lot for the code review.